copter/source/docs/traditional-helicopter-tuning.rst

```diff
@@ -315,86 +260,28 @@ discrepancy between desired and actual pitch and roll attitude when the
-drifts make small adjustments to the :ref:`SERVO1_TRIM<SERVO1_TRIM>` , :ref:`SERVO2_TRIM<SERVO2_TRIM>` and :ref:`SERVO3_TRIM<SERVO3_TRIM>` .
-The chances of getting the swashplate perfectly level during bench setup is very
-low and this dynamic tuning is needed to trim the helicopter. If it requires
-large deviation from your original ``SERVOx_TRIM`` values it is likely you have a CG
-problem, or your initial setup when leveling the swashplate was not very
-accurate.
-
-Your helicopter is now trimmed properly. This trimming procedure makes the
-difference between a helicopter that is difficult to handle vs one that flies
-with true scale quality and handling. 
-
-Adjusting I-gains For High-Speed Autonomous Flight
-==================================================
-Prepare a mission with your ground station software that will fly the 
-helicopter, preferably in a figure-8 pattern to make both right and left turns,
-at a speed of 6 m/s. Fly the helicopter on this mission, pull the logs from the
-microSD card and look at the AHRS desired vs actual pitch, roll and yaw
-attitudes in dynamic flight. They should track within 1-2 degrees. If they do
-not, increase the ``ATC_RAT_xxx_I`` value for that axis until they do.
-
-Now, fly the same mission, but at higher speed of 9-10 m/s, and analyze the logs
-the same way. Make further adjustments to the I-gains and IMAX values as
-required. It is not clear what I-gain values will be required as no two
-helicopters are the same. But I-gain values from 0.25 - 0.38 are common in pitch
-and roll, and 0.18 - 0.30 in yaw. IMAX values of 0.40 - 0.45 are common, however
-refer to the 'Setting the I gain, IMAX, and ILMI' section on how to determine
-what the IMAX value should be.
+Tuning a Flybar Equipped Head
+=============================
+
+With a flybar head, where the linkage rate is normally lower, it is recommended
+to start with 0.22 FF for both pitch and roll and you will likely have to go
+higher with FF. But for a flybarless head, FF shouldn't be more than 0.22 
+unless you have really really slow servos or slow linkage rate. With all 
+helicopters, the FF gain compensates for differences in servo and linkage
+speed. 
+
+**Note on Tuning Flybar Helicopers** - for flybar heads the flybar is a
+mechanical version of the rate PID loop. So flybar is tuned using only FF in
+pitch and roll. The rate D and P gains mentioned in the next topic below are
+left set to zero for flybar. I-gain, IMAX, and ILMI are tuned just like FBL.
+Below is a plot of beginning a tune on a flybar helicopter using the starting
+setting of :ref:`ATC_RAT_RLL_VFF<ATC_RAT_RLL_VFF>` = 0.22. The graph shows the aircraft's response to
+the rate request of the attitude controller is low, meaning the FF value must
+be increased to achieve proper rate response. For all flybar helicopters be sure
+to set :ref:`H_FLYBAR_MODE<H_FLYBAR_MODE>` = 1
```

Comment:
Flybar mode is a bit misleading.  It only provides direct passthrough for acro mode.  In stabilize and all other modes, it uses the normal command model and PID archetecture.

Reply:
reworked this